02_bubble/shuutech/main.go

```diff
@@ -0,0 +1,27 @@
+package main
+
+import (
+	"fmt"
+)
+
+func bubble(s []int) []int {
+	n := len(s)
+	var swapped bool
+	for ok := true; ok; ok = swapped {
+		swapped = false
+		for i := 1; i < n; i++ {
+			if s[i-1] > s[i] {
+				a := s[i-1]
```

Comment:
You don't need temporary variables a & b

i.e.

```s[i-1], s[i] = s[i], s[i-1]```

Reply:
i have fixed this - please review